docs/framework/network-programming/tls.md

```diff
@@ -15,34 +15,57 @@ helpviewer_keywords:
+.NET Framework supports the use of the Transport Layer Security (TLS) protocol to secure network communications.
 
-To ensure .NET Framework applications remain secure, the TLS version should **not** be hardcoded. .NET Framework applications should use the TLS version the operating system (OS) supports.
+## What is Transport Layer Security (TLS)?
 
-This document targets developers who are:
+> [!WARNING]
+> TLS 1.0 and 1.1 has been deprecated by [RFC8996](https://datatracker.ietf.org/doc/rfc8996/). This document will cover TLS 1.2 and TLS 1.3.
+
+The Transport Layer Security (TLS) protocol is an industry standard designed to help protect the privacy of information communicated over the Internet. [TLS 1.3](https://tools.ietf.org/html/rfc8446) is a standard that provides security improvements over previous versions. This article presents recommendations to secure .NET Framework applications that use the TLS protocol.
+
+## Who can benefit from this document?
 
 - Directly using the <xref:System.Net> APIs (for example, <xref:System.Net.Http.HttpClient?displayProperty=nameWithType> and <xref:System.Net.Security.SslStream?displayProperty=nameWithType>).
 - Directly using WCF clients and services using the <xref:System.ServiceModel?displayProperty=nameWithType> namespace.
 
-Consider the following recommendations:
+## TLS support in .NET Framework
+
+Since the .NET Framework is dependent on `Schannel` on Windows, which versions can be negotiated and which version will be used depends on the operating system.
+
+Here is an updated example table showing the highest supported TLS version for different combinations of operating system versions and .NET Framework target versions:
+
+| .NET Framework Target Version | Windows 10 | Windows 11 |
+|-------------------------------|------------|------------|
+| 3.5                           | TLS 1.2    | TLS 1.2    |
+| 4.6.2                         | TLS 1.2    | TLS 1.3    |
+| 4.7                           | TLS 1.2    | TLS 1.3    |
+| 4.7.1                         | TLS 1.2    | TLS 1.3    |
+| 4.7.2                         | TLS 1.2    | TLS 1.3    |
+| 4.8                           | TLS 1.2    | TLS 1.3    |
+| 4.8.1                         | TLS 1.2    | TLS 1.3    |
+
+For more information see [TLS protocol version support in Schannel](https://learn.microsoft.com/en-us/windows/win32/secauthn/protocols-in-tls-ssl--schannel-ssp-#tls-protocol-version-support).
 
-- For TLS 1.2, target .NET Framework 4.7 or later versions on your apps, and target .NET Framework 4.7.1 or later versions on your WCF apps.
-- For TLS 1.3, target .NET Framework 4.8 or later.
+## Recommendations
+
+- For TLS 1.3, target .NET Framework 4.6.2 or later.
```

Comment:
I think 4.6 is out of support, right @karelz? While there may be way how to make it work (probably) it probably is not best practice.

Reply:
4.6.2 is still supported, see: https://learn.microsoft.com/en-us/lifecycle/products/microsoft-net-framework